### PR TITLE
Integrate flask python with AWS

### PR DESCRIPTION
**Pull request by kimyx**
This fork integrates the AWS Elastic Beanstalk setup into the VECTOR/api directory. These changes are running successfully on the SWxTREC account at AWS. It required some renaming of the original python flask application.

Next step will be changing the internals of the python flask to call matlab code on the EB server.

### CODE REVIEW

File: api/application.py [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 import subprocess
 
-app = Flask(__name__)
-CORS(app)
+application = Flask(__name__)
```
Comment (greglucas) [inline api/application.py]: I saw `@app` used basically everywhere online, including Flask's documentation.
https://flask.palletsprojects.com/en/1.1.x/appcontext/

Was this part of the required renaming you mention in the main PR comment?

File: .gitignore [hunk 2]
```diff
@@ -30,3 +30,4 @@ codegen/
 
 # Octave session info
 octave-workspace
+
```
Comment (greglucas) [inline .gitignore]: Are you using an auto-code formatter? My linter always prefers a newline at the end of the files too, but @jennyknuth actually removed the extra newline at the end in her last commit :) Maybe a Python/javascript preference difference. I'm indifferent either way.

File: api/application.py (hunk 1 above)
Comment (kimyx) [reply api/application.py]: Yes, EB requires the file to be named application.py and the application variable to be named application. There's a way to use a "non-standard" filename but it seemed better to go with their defaults.

File: .gitignore (hunk 2 above)
Comment (kimyx) [reply .gitignore]: I use an auto-formatter for python, but I've never worried about the eof newline issue for .gitignore files. I'm not picky about this particular topic.

File: api/application.py (hunk 1 above)
Comment (kimyx) [reply api/application.py]: I had to write all the .ebextensions, hence why they are being checked in.

There's another .elasticbeanstalk directory that has other kinds of one-time/prepopulated settings that isn't being committed.

File: .gitignore (hunk 2 above)
Comment (jennyknuth) [reply .gitignore]: Ditto, but I might have just added this particular commit directly from the GitHub editor, so not sure what it did. If this persists as a problem, and we have competing editors, we can address it.

**Pull request merged**